data/dev.geopjr.Collision.metainfo.xml.in

```diff
@@ -50,4 +50,13 @@
     <release version="1.0.1" date="2021-02-11" />
   </releases>
   <content_rating type="oars-1.1" />
+  <translation type="gettext">dev.geopjr.Collision</translation>
+  <recommends>
+    <control>keyboard</control>
+    <control>pointing</control>
+    <control>touch</control>
+  </recommends>
+  <requires>
+    <display_length compare="ge">360</display_length>
+  </requires>
```

Comment:
Not sure if it should be added yet. While it is true, GNOME Software's description of "Works on phones, tablets and desktops" might be a bit misleading.

As per #19, there are only x86_64 builds of Crystal available (and as mentioned on a flathub issue review, aarch64 is [tier 2](https://crystal-lang.org/reference/master/platform_support.html#tier-2) ("Tier 2 platforms can be thought of as 'expected to build'. Automated tests are not run so it’s not guaranteed to produce a working build, [...]")).

Since current non-desktop devices are mostly non-x86_64, the caption of "Works on phones, tablets and desktops" doesn't sound completely true. But at the same time, the tags are not wrong.

Open for discussion as always!

Reply:
It makes sense to leave these out then, I saw that issue a while ago and I didn't remember about it 😅 Although Collision works great with small window sizes and probably for touch screens as well. Also they're broken for Flatpaks, so it isn't very useful to have them if they aren't going to show up anyway.

Initially this PR was meant to fix apps.gnome.org showing Collision not available for my language and that only requires translation tag.